docs/pages/components/TwoColumnPanel.mdx

```diff
@@ -0,0 +1,310 @@
+---
+title: TwoColumnPanel
+description: A container component intended to provide a navigable list of items that can be collapsed if not needed.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/TwoColumnPanel/TwoColumnPanel.tsx
+---
+
+import { useState } from 'react'
+import {
+  TwoColumnPanel,
+  ColumnLeft,
+  ColumnRight,
+  ColumnHeader,
+  ColumnList,
+  ColumnGroupHeader,
+  Breadcrumb,
+  BreadcrumbLink,
+  BreadcrumbItem
+} from '@deque/cauldron-react'
+
+```jsx
+import {
+  TwoColumnPanel,
+  ColumnLeft,
+  ColumnRight,
+  ColumnHeader
+} from '@deque/cauldron-react'
+```
+
+## Examples
+
+### Default
+
+`TwoColumnPanel` takes in a set of columns, `ColumnLeft` and `ColumnRight` with the left column containing the list of items to choose from and the right column intended to be the currently selected item. It is expected that accessible names are provided for both of the left/right column via `aria-label` or `aria-labelledby`.
+
+```jsx example
+function TwoColumnPanelExample() {
+  const [selected, setSelected] = useState(1);
+
+  const items = [
+    {
+      id: 1,
+      name: 'One',
+      description: 'Short description of one',
+      contents: 'Content for One'
+    },
+    {
+      id: 2,
+      name: 'Two',
+      description: 'Short description of two',
+      contents: 'Content for Two'
+    },
+    {
+      id: 3,
+      name: 'Three',
+      description: 'Short description of three',
+      contents: 'Content for Three'
+    }
+  ];
+
+  const selectedItem = items.find(item => item.id === selected);
+  const selectItem = id => event => {
+    event.preventDefault();
+    setSelected(id);
+  };
+
+  return (
+    <TwoColumnPanel>
+      <ColumnLeft aria-labelledby="sidebar">
+        <ColumnHeader id="sidebar">Items</ColumnHeader>
+        <nav aria-label="Sidebar navigation">
```

Comment:
"navigation" seems redundant with the implicit role of "navigation" for `<nav />` elements

```suggestion
        <nav aria-label="Sidebar">
```

Sidebar itself doesn't seem like great name here either but these are contrived examples so I don't think it is a huge deal. Ideally there would be some kind of descriptive heading above the two column panel (like they always do when we use this component in our apps) that we could make the nav `aria-labelledby`

Reply:
Yeah I think I agree with your reasoning. I've updated the label to point to the header for both instances as that's more likely how we'd use it in practice.